ionoscloud/resource_server.go

```diff
@@ -674,7 +679,7 @@ func resourceServerCreate(ctx context.Context, d *schema.ResourceData, meta inte
 				if sentFirstNic.Entities != nil && sentFirstNic.Entities.Firewallrules != nil && sentFirstNic.Entities.Firewallrules.Items != nil {
 					sentRules := *sentFirstNic.Entities.Firewallrules.Items
 					foundRules := *foundFirstNic.Entities.Firewallrules.Items
-					orderedRuleIds = firewallSvc.ExtractOrderedFirewallIds(foundRules, sentRules)
+					orderedRuleIds = cloudapifirewall.ExtractOrderedFirewallIds(foundRules, sentRules)
```

Comment:
@cristiGuranIonos from now on I would suggest keeping changes separately for more clarity, but since you already make these changes I suggest changing the name of the PR or the commit to also include information about the changes related to firewall service renaming.